rel/files/mongooseim.cfg

```diff
@@ -636,7 +636,8 @@
 {services,
     [
         {service_admin_extra, [{submods, [node, accounts, sessions, vcard, gdpr,
-                                          roster, last, private, stanza, stats]}]}
+                                          roster, last, private, stanza, stats]}]},
+        {mongoose_user_stats, []}
```

Comment:
IMO since all modules follow the convention with `mod_` prefix, we should do the same with `service_` for services.